### DISCUSSION TITLE
Assets aren't loading correctly

### DISCUSSION THREAD

**Discussion by dannyghub** [Laravel-Backpack/community-forum#884] [Q&A (Help)] [answered]
Hi,
In my local environment the assets load correctly, but in my staging env it seems absolute path is being appended to the domain

![image](https://github.com/Laravel-Backpack/community-forum/assets/63340376/ece6210c-412f-4c64-a48c-5f4c2c00afb9)

I have tried reinstalling in the staging env `php artisan backpack:install`
I also checked that my .env has the correct APP_URL
And I've tried flushing cache with `php artisan cache:clear` but the problem persists.

Any idea what could cause this?

**Comment by pxpm** [comment]
Hey @dannyghub what basset version are you using ? You can check versions with `php artisan backpack:version`

Can you also try a `php artisan basset:clear && php artisan basset:cache` ?

Cheers

**Reply by dannyghub** [reply]
Hey @pxpm , 

local:
```
### PHP VERSION:
PHP 8.2.12 (cli) (built: Oct 26 2023 17:33:49) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.12, Copyright (c), by Zend Technologies
    with Xdebug v3.3.0, Copyright (c) 2002-2023, by Derick Rethans

### LARAVEL VERSION:
10.47.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.4
backpack/crud: 6.7.0
backpack/generators: v4.0.4
backpack/theme-coreuiv4: 1.1.1

```
staging: 
```
### PHP VERSION:
PHP 8.2.10-2ubuntu1 (cli) (built: Sep  5 2023 14:37:47) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.10, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.10-2ubuntu1, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.47.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.4
backpack/crud: 6.7.0
backpack/generators: v4.0.4
backpack/theme-coreuiv4: 1.1.1
```

**Reply by dannyghub** [reply]
Trying `php artisan basset:clear && php artisan basset:cache` did not resolve it sadly

```
Clearing basset '/storage/app/public/basset'
Done
Looking for bassets under the following directories:
 - resources/views (15 blade files)
 - vendor/backpack/crud/src/resources/views (139 blade files)
 - vendor/backpack/theme-coreuiv4/resources/views (34 blade files)

Found 57 bassets in 188 blade files. Caching:
 57/57 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

Done in 0.04s
```
But the error stays the same

**Reply by o15a3d4l11s2** [reply]
This `/var/www` inside the url path makes me think there might be something strangely configured in the storage or filesystem. Can you verify that your storage is linked? Do you have some custom logic for the public disk or other custom disk locations?

**Reply by dannyghub** [reply]
Yeh it's linked with `php artisan storage:link` and the FE assets load correctly, it's a fairly vanilla Laravel install, nothing wacky going on.

**Reply by o15a3d4l11s2** [reply]
Can you please check [this discussion](https://github.com/Laravel-Backpack/basset/issues/88), it seems your problem is similar.

**Reply by dannyghub** [reply, marked as answer]
Thanks @o15a3d4l11s2 , it didnt directly resolve but I found the cause.

If I ran `php artisan basset:clear && php artisan basset:cache` It would say everything was OK, but actually there must be a silent fail somewhere, because if I just run `php artisan basset:clear` and try visit the dashboard, I got a 500 permissions issue where basset is trying to store the assets deep in the storage path. So changing permissions in the storage path and refreshing, everything is now fine.

Thank you both for your help.

**Reply by o15a3d4l11s2** [reply]
Glad you solved it!
